Create page for the Merge Data Challenge (#7182)
ethereum/ethereum-org-website: src/content/community/grants/merge-data-challenge/index.md

---
title: Merge data challenge
description: Instructions, deadlines, and prize information for the Merge data challenge
lang: en
sidebar: true
sidebarDepth: 2
---

# The Merge data challenge {#the-merge-data-challenge}

_Up until four weeks after the Merge, the Ethereum Foundation is sponsoring a Merge data analysis and data visualization blog post challenge. Here are all the details you need._

<Divider />

Calling all Ethereans, data scientists, data engineers, data visualizers, developers, and anyone interested in digging into Ethereum data!

The [Merge](https://ethereum.org/en/upgrades/merge/) is coming, providing a more secure and sustainable home for Ethereum -- as well as more data, lots and lots of data. And **the Ethereum community needs your help to make sense of it all.**

- _What new visualizations help provide insight into proof-of-stake Ethereum?_

> - _How do the Consensus-Layer and Execution-Layer interact? Are there differences across client pairs in communication patterns, efficiency, etc? Are their good places to optimise?_

Reviewer: ```suggestion
- _How do the Consensus-Layer and Execution-Layer interact? Are there differences across client pairs in communication patterns, efficiency, etc? Are there good places to optimise?_
```

- _Did the Merge affect core Beacon Chain activity -- attestation performance, blocks missed, sync committees?_
- _Did user activity noticeably change after the Merge? What about MEV?_
- _What new tools can you build to collect and analyze data in the post-Merge network?_

## The challenge {#the-challenge}

**Document your best Merge data insights in the most readable blog post possible – for prizes!**

The Ethereum Foundation is running this challenge because there's a lot to learn and discover from the Merge mainnet activity. Your findings will give the Ethereum community -- from beginners to researchers and client developers -- important insight into the Merge.

Get started now if you want to gather date from both _before_ and _after_ the big event!

 <Emoji svg text=":rocket:" />

### How to submit {#how-to-submit}

Anyone is free to submit, here's how:

1.  Collect and analyze Merge data
    - Either with existing tools
    - Or for the extra ambitious, build your own tool and tell us about it!
2.  Detail out your work in a blog post
3.  Submit!

<!-- TODO: link to new submission form -->

> <ButtonLink to="https://esp.ethereum.foundation/">Submit blog post</ButtonLink>

Reviewer: ```suggestion
<ButtonLink to="https://esp.ethereum.foundation/merge-data-challenge">Submit blog post</ButtonLink>
```

---
title: Merge data challenge
description: Instructions, deadlines, and prize information for the Merge data challenge
lang: en
sidebar: true
sidebarDepth: 2
---

# The Merge data challenge {#the-merge-data-challenge}

_Up until four weeks after the Merge, the Ethereum Foundation is sponsoring a Merge data analysis and data visualization blog post challenge. Here are all the details you need._

<Divider />

> ![Two people holding hands in field with ethereum logo](./merge-data-challenge.jpg)

Reviewer 2: This is decorative, so it should have an empty alt.

```suggestion
![](./merge-data-challenge.jpg)
```

Author: Sorry I missed this. I don't, but tomo might

> ![Two people holding hands in field with ethereum logo](./merge-data-challenge.jpg)

Reviewer 2: Just noting this feels like an unusual place to have an image. But, it's probably the best solution as we don't have a template to support this (yet).

Calling all Ethereans, data scientists, data engineers, data visualizers, developers, and anyone interested in digging into Ethereum data!

> The [Merge](https://ethereum.org/en/upgrades/merge/) is coming, providing a more secure and sustainable home for Ethereum -- as well as more data, lots and lots of data. And **the Ethereum community needs your help to make sense of it all.**

Reviewer 2: ```suggestion
[The Merge](https://ethereum.org/en/upgrades/merge/) is coming, providing a more secure and sustainable home for Ethereum -- as well as more data, lots and lots of data. And **the Ethereum community needs your help to make sense of it all.**
```

- _What new visualizations help provide insight into proof-of-stake Ethereum?_
- _How do the Consensus-Layer and Execution-Layer interact? Are there differences across client pairs in communication patterns, efficiency, etc? Are their good places to optimise?_
- _What, if anything, changed on the network at the point of the Merge -- block propagation times, p2p connections, transaction mempool performance, etc?_
- _Did the Merge affect core Beacon Chain activity -- attestation performance, blocks missed, sync committees?_
- _Did user activity noticeably change after the Merge? What about MEV?_
- _What new tools can you build to collect and analyze data in the post-Merge network?_

## The challenge {#the-challenge}

**Document your best Merge data insights in the most readable blog post possible – for prizes!**

The Ethereum Foundation is running this challenge because there's a lot to learn and discover from the Merge mainnet activity. Your findings will give the Ethereum community -- from beginners to researchers and client developers -- important insight into the Merge.

Get started now if you want to gather date from both _before_ and _after_ the big event!

>  <Emoji svg text=":rocket:" />

Reviewer 2: ```suggestion
 <Emoji text=":rocket:" size={1} />
```

### How to submit {#how-to-submit}

Anyone is free to submit, here's how:

1.  Collect and analyze Merge data
    - Either with existing tools
    - Or for the extra ambitious, build your own tool and tell us about it!
2.  Detail out your work in a blog post
3.  Submit!

<!-- TODO: link to new submission form -->

<ButtonLink to="https://esp.ethereum.foundation/">Submit blog post</ButtonLink>

_You may submit more than one blog post! So long as each posts focuses on a different visualisation or piece of analysis_

In addition to the prompt questions above, here's a [wishlist of data analysis avenues to explore](https://notes.ethereum.org/@djrtwo/merge-data-comp-wish).

> A group of Ethereum community members will evaluate your submissions and awards will go to the top blog posts <Emoji svg text=":trophy:" />

Reviewer 2: ```suggestion
A group of Ethereum community members will evaluate your submissions, and awards will go to the top blog posts <Emoji text=":trophy:" size={1} />.
```

> <h2>Deadline <Emoji svg text=":alarm_clock:" /></h2>

Reviewer 2: ```suggestion
## Deadline <Emoji text=":alarm_clock:" size={1} /> {#deadline}
```

The deadline for submissions is **four weeks after the merge** (precise date TBD).

> <h2>Prizes <Emoji svg text=":money_with_wings:" /></h2>

Reviewer 2: ```suggestion
## Prizes <Emoji text=":money_with_wings:" size={1} /> {#prizes}
```

Up to **USD $30,000**

_Entries must be considered sufficiently impactful/insightful by the community judging team to be eligible for rewards._

## Requirements {#requirements}

- Blog post must be in English.
  - There are no length requirements.
- Blog posts must be public and original.

> - Data analysis or visualization must concern [Merge](https://ethereum.org/en/upgrades/merge/) network data.

Reviewer 2: ```suggestion
- Data analysis or visualization must concern [Merge](/upgrades/merge/) network data.
```

- Write new tools to gather data.
- Crunch some numbers and spin up some graphs.
- Publish your analysis or visualization!

## Helpful resources {#helpful-resources}

- [Merge Overview](https://ethereum.org/en/upgrades/merge/)
- Specs
  - [Consensus Layer specs](https://github.com/ethereum/consensus-specs)
  - [Execution Layer specs](https://github.com/ethereum/execution-specs)
  - [EIP 3675](https://eips.ethereum.org/EIPS/eip-3675) & [EIP 4399](https://eips.ethereum.org/EIPS/eip-4399)
  - [Engine API](https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md)
- [Mega Merge Resource List](https://notes.ethereum.org/@MarioHavel/merge-resources)
- [Ethereum Roadmap FAQ](https://github.com/timbeiko/eth-roadmap-faq)
- [Ethereum Energy Consumption](https://ethereum.org/en/energy-consumption/)
- [Eth R&D](https://discord.gg/VmG7Uxc) _– Discord_
- [ethstaker](https://invite.gg/ethstaker) _– Discord_

## Support {#support}

> For any general support questions about your submission, please email datachallenge@ethereum.org.

Reviewer 2: ```suggestion
For any general support questions about your submission, please email [datachallenge@ethereum.org](mailto:datachallenge@ethereum.org).
```
